src/collections/_documentation/common/getting_started/getting_started_template.md

```diff
@@ -0,0 +1,124 @@
+<!--
+Guideline: This page is common to all SDKs; add the name of the SDK you're documenting as well as the code samples as instructed. If you have questions, please ask Fiona or Daniel. 
+
+**The objective for this page is that a developer can quickly complete an ideal integration.**
+-->
+
+Add Sentry's <!--SDK name--> SDK to automatically report errors, exceptions, and rejections in your application. 
+
+On this page, we provide concise information to get you up and running quickly. To learn more, use the links to in-depth content.
+
+In the right place? We also offer documentation for:
+<!-- 
+Guideline: Add icons for related languages/frameworks.
+-->
+What's covered on this page:
+
+- [Install](#install)
+- [Initialize](#initialize)
+- [Verify Setup](#verify-setup)
+- [Capture Errors](#capture-errors)
+    1. [Enrich Error Data](#enrich-error-data) 
+    2. [Set the Release Version](#set-release-version)
+    
+<!--
+Guideline: Add any step to set up information that's specific to the SDK; for example, in the JavaScript example, this heading is for Source Maps. Then add the link to that heading
+-->
+
+- [Monitor Performance](#monitor-performance)
+
+## Install {#install}
+
+1.  If you already have an account, log in to Sentry. If not, establish your account:
+    1. Use your Google, GitHub, or Azure Devops account
+    2. Or, provide the basics: Your name, email, and organization, then set your password.
+2. Select <SDK> as the platform you're working with
+3. Name your project
+
+*Projects* in Sentry scope events to a distinct application in your organization, and assign responsibility and ownership. 
+
+4. Choose your installation method:
+
+<!--
+Guideline: Add SDK specific installation information
+-->
+
+## Initialize {#initialize}
+
+`init` Sentry as soon as possible:
+<!--
+Guideline: Add init code example for this SDK 
+-->
+Once this is done, all unhandled exceptions are automatically captured by Sentry. 
+
+**Important: Record your DSN.** The *DSN* (Data Source Name) tells the SDK where to send the error, associating errors with the project you just created. If you forget your DSN, log in to Sentry to obtain the project DSN from your organization's *Settings -> Projects -> Client Keys (DSN)* in the Sentry web UI. The DSN includes the protocol, public key, server address, and project identifier, using this format:
+
+`https://<Public Key>@<Sentry Server Address>/<Project Identifier>`
+
+## Verify Setup {#verify-setup}
+
+Verify your setup by intentionally sending an error that breaks your application. Sentry will send an alert to the email address configured on [Sentry.io](http://sentry.io/) notifying you about the event.
```

Comment:
I would not tell them about the alert thing, as that's highly variable

Reply:
just tell them to go to the UI to identify the error. it may take a few minutes.

Reply:
Got it.